src/redux/constant.js

```diff
@@ -0,0 +1,3 @@
+export const SPRITE_SIZE = 40
```

Comment:
Hmm. I'm not sure if these map sizes are necessary, either. We should be able to make the size of the map dynamic, after all.

Up to you whether you want to keep them in. I'll likely not use them in any case.